report_artifact_test.go

```diff
@@ -141,13 +141,12 @@ func TestUpload(t *testing.T) {
 		{
 			name: "NoRemotePath",
 			artifact: &TestArtifact{
-				Bucket:    "bucket",
+				Bucket:    "s3Name",
```

Comment:
Sorry, didn't specify clearly but this is a variable `s3Name` so no need for the quotation marks.

Reply:
hahaha I absolutely should have caught on there...

:bushes: